source/common/upstream/upstream_impl.cc

```diff
@@ -1148,13 +1148,17 @@ void ClusterImplBase::onPreInitComplete() {
   }
   initialization_started_ = true;
 
-  ENVOY_LOG(debug, "initializing {} cluster {} completed",
+  ENVOY_LOG(debug, "pre-initializing {} cluster {} completed",
             initializePhase() == InitializePhase::Primary ? "Primary" : "Secondary",
             info()->name());
   init_manager_.initialize(init_watcher_);
 }
 
 void ClusterImplBase::onInitDone() {
+  ENVOY_LOG(debug, "initializing {} cluster {} completed",
```

Comment:
will this be confusing for users since it's the same log message we used to have in `onPreInitComplete()`? should I change it slightly?